tests/unit/via/services/url_details_test.py

```diff
@@ -94,7 +94,7 @@ def test_it_when_youtube_disabled(self, youtube_service, GoogleDriveAPI, svc):
         result = svc.get_url_details(sentinel.youtube_url)
 
         youtube_service.get_video_id.assert_not_called()
-        assert result == ("dummy", 200)
```

Comment:
Dummy gets parsed as the default according to the spec, text/plain.

Using a more explicit testing mimetype.

Reply:
Would `sentinel.content_type` work here? Sentinels are often a bit more explicit than strings that a test sentinel value is being used

Reply:
Not without mocking the content type parsing which I reckon will make the test less usueful.